tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn.py

```diff
@@ -175,8 +175,12 @@ def _process_pytorch_outputs(self, raw_outputs, identifiers, frame_meta):
         if classes is None:
             classes = np.ones(len(boxes), np.uint32)
 
-        valid_detections_mask = classes > 0
-        classes = classes[valid_detections_mask]
+        if self.scores_out:
+            valid_detections_mask = classes > 0
+            classes = classes[valid_detections_mask]
+        else:
+            valid_detections_mask = np.sum(boxes, axis=1) > 0
+            classes = classes[valid_detections_mask] + 1
```

Comment:
not sure that it is necessary, we prefer to align classes in annotation with model labels, there is option which allow convert dataset without background label

Reply:
Fixed.